locallib.php

```diff
@@ -113,25 +113,46 @@ public function get_settings(MoodleQuickForm $mform) {
 
         require_once($CFG->dirroot . '/mod/assign/submission/maharaws/lib.php');
 
-        $mform->addElement('text', 'assignsubmission_maharaws_url', get_string('url', 'assignsubmission_maharaws'), array('maxlength' => 255, 'size' => 50));
+        $forceGlobalCredentials = get_config('assignsubmission_maharaws','force_global_credentials');
```

Comment:
@synesthesia  would be nice if you can follow Moodle coding styles for the var names see https://docs.moodle.org/dev/Coding_style#Variables